File: java/src/main/java/mx/iteso/ut/TortillaMaiz.java
package mx.iteso.ut;
/**
 * clase tortilla mazi.
 */
public class TortillaMaiz implements Tortilla {
    /**
     *
     * @return esta tostada?.
     */
    public boolean isToasted() {
        return false;
    }

    /**
     *
     * @return temperatura tostada.
     */
    public int getToastTemperature() {
        return 0;
    }

    /**
     *
     * @return temperatura ahorita.
     */
    public int getCurrentTemperature() {
        return 0;
    }

    /**
     *
     * @param temp temperatura de ahorita.
     */
    public void setCurrentTemperature(final int temp) {

    }

    /**
     *
     * @param toasted esta tostada.
     */
    public void toast(final boolean toasted) {

    }
}

File: python/TortillaMaiz.py
class TortillaMaiz:

    def __init__(self):
        self.toasted = None
        self.currentTemperature = None
        self.toastTemperature = 40

    def isToasted(self):
        return self.toasted

    def getToastTemperature(self):
        return self.toastTemperature

    def getCurrentTemperature(self):
        return self.currentTemperature

    def setCurrentTemperature(self, temp):
        self.currentTemperature = temp

    def toast(self, toasted):
        self.toasted = toasted
